### ISSUE TITLE
Doesn't detect whitespaces within properties

### ISSUE THREAD

**Issue by GaetanBt** [CSSLint/csslint#554]
Hey there, 

Sorry if someone opened an similar issue but I didn't find any.

CSSLint doesn't seem to detect whitespaces within a property value.

For example :

```
.test {
  cursor: p ointer;
}
```

Thanks :)
